File: Php/app/Urls/PlayerAddUrlModel.php
<?php
namespace app\Urls{

	use app\RouteFormats;
	use app\Urls\BaseClasses\HomegameUrlModel;
	use entities\Homegame;

	class PlayerAddUrlModel extends HomegameUrlModel{

		public function __construct(Homegame $homegame){
			parent::__construct(RouteFormats::playerAdd, $homegame);
		}

	}

}

File: Web/Models/UrlModels/PlayerAddUrlModel.cs
using Core.Classes;
using Web.Routing;

namespace Web.Models.UrlModels{
    public class PlayerAddUrlModel : HomegameUrlModel{

		public PlayerAddUrlModel(Homegame homegame)
            : base(RouteFormats.PlayerAdd, homegame)
        {
		}

	}

}